### PR TITLE
Make about page responsive

### LINKED ISSUE #418: About page is not responsive
About page on mobile looks pretty bad, let's make it responsive like the other pages

Screen Shot 2019-10-12 at 8 50 51 PM

## Expected Behavior

About page works responsively like the other pages.

## Current Behavior

About page text looks bad and unreadable.

### PR DESCRIPTION
**Pull request by PabloPap**
This PR changes the sass files of the about page so it's now responsive
fixes #418

### CODE REVIEW

File: src/components/About/How/How.scss [hunk 1]
```diff
@@ -31,6 +32,15 @@
         margin-bottom: $margin-s;
         margin-left: $margin-xs;
       }
+
+      p {
+        text-align: justify;
```
Comment (javpet) [inline src/components/About/How/How.scss]: Could you please remove `justify` and just use left-aligned text here?

File: src/components/About/Vision/Vision.scss [hunk 2]
```diff
@@ -32,8 +32,26 @@
     text-align: center;
   }
 
+  p {
+    text-align: justify;
```
Comment (javpet) [inline src/components/About/Vision/Vision.scss]: Could you please remove `justify` and just use left-aligned text here?

**Review by javpet — APPROVED** [review]
Great work @PabloPap! Thanks for your contribution! If you want to follow along later we'll have other issues as well to work on like Home page, Adding artists page. 🙂

**Comment by aonomike** [comment]
Thanks @PabloPap for working on this...

**Comment by PabloPap** [comment]
@javpet , @aonomike thanks for your collaboration, I'll keep an eye for other issues

**Pull request merged**